bazel/cc_toolchains/clang_configuration.bzl

```
@@ -63,6 +63,10 @@ def _compute_clang_cpp_include_search_paths(repository_ctx, clang, sysroot):
     Returns the resulting paths as a list of strings.
     """
 
+    # Create an empty temp file for Clang to use
+    if repository_ctx.os.name.lower().startswith("windows"):
+        repository_ctx.file('_temp', '')
```

```suggestion
    # Read in an empty input file. If we are building from
    # Windows, then we create an empty temp file. Clang on Windows does not like it
    # when you pass a non-existent file.
    if repository_ctx.os.name.lower().startswith("windows"):
        repository_ctx.file('_temp', '')
        input_file = repository_ctx.path('_temp')
    else:
        input_file = "/dev/null"
```

I find Python's ternary syntax a little hard to read, so I find this kind of approach clearer (it also puts the logic in one place). Note my comment wrap needs to be fixed, but I figure that's easier in a real code editor.

---

I agree, keeping the platform-specific login in one place like this makes more sense and is much clearer. Thank you for the feedback.